src/app/base/components/FilterAccordion/FilterAccordion.tsx

```diff
@@ -104,6 +105,7 @@ const sortByFilterKey = (
 };
 
 const FilterAccordion = <I, PK extends keyof I>({
+  ariaLabel,
```

Comment:
If not using a props spread you can pass this down like so:
```tsx
const FilterAccordion = <I, PK extends keyof I>({
  disabled,
  (...)
  "aria-label": ariaLabel
/>

  return (
    <ContextualMenu
      aria-label={ariaLabel}
```

Alternatively when using a props spread: 


```tsx
const FilterAccordion = <I, PK extends keyof I>({
  disabled,
  (...)
  {...props}
/>

  return (
    <ContextualMenu
      aria-label={props["aria-label"]}
      ```